SystemServices/SystemServices.cpp

```diff
@@ -5350,13 +5345,46 @@ namespace WPEFramework {
            if (WDMP_SUCCESS == wdmpstatus) {
                 migrationstatus = param.value;
                 LOGINFO("Current ENTOS Migration Status is: %s\n", migrationstatus.c_str());
-                response["MigrationStatus"] = migrationstatus;
+                response["migrationStatus"] = migrationstatus;
                 status = true;
             }
             else {
                 LOGINFO("Failed to get RFC parameter for Migration Status \n");
             }
-            returnResponse(status);
-       }
+         return (status ? WPEFramework::Core::ERROR_NONE : 1100);
+        }//end of getMigrationStatus method
+       /*
+         * @brief This function updates plugin API error text.
+         * This method is called by thunder right after Plugin API.
+         * @param1[in]: Context
+	 * @param2[in]: method
+         * @param3[in]: parameters
+	 * @param4[in]: errorcode
+         * @param5[out]: errormessage
+         * @return: Core::<StatusCode>
+         */
+        uint32_t SystemServices::OnJSONRPCError(const Core::JSONRPC::Context&, const string& method, const string& parameters, const uint32_t errorcode, string& errormessage) {
+           if((method == _T("getMigrationStatus")) && (errorcode == 1100) ) { //ERROR_GETVALUE_FAILED - 1100
+               std::stringstream message;
+               message <<_T("Value retrieval failed");
```

Comment:
All the message are static text, so the text can be directly copied to "errormessage", no need to define a stringstream.